apps/meteor/app/livechat/server/lib/departmentsLib.ts

```diff
@@ -25,7 +26,7 @@ export async function saveDepartment(
 	departmentData: LivechatDepartmentDTO,
 	departmentAgents?: {
 		upsert?: { agentId: string; count?: number; order?: number }[];
-		remove?: { agentId: string; count?: number; order?: number };
+		remove?: { agentId: string; count?: number; order?: number }[];
```

Comment:
looks like we don't even use `remove` param right?

Reply:
Apparently we dont. I'm still checking if we can remove this one cause endpoint does allow it.
The problem here is that when i checked on the types, this `remove` should be an array, but this was not, and when i updated ajv and other places to reflect this was an array, then this complained 😢 

(If we can remove, i'll remove it on next major most likely, or use this more and remove any meteor method that can still be in use)